src/main/webapp/js/jquery.emoji.js

```diff
@@ -0,0 +1,9 @@
+// A very simple jQuery wrapper for js-emoji
```

Comment:
Wrapper that does what?

It looks like it adds a jquery plugin?

Reply:
This was included in the `js-emoji` library, I just forgot to delete it from our `js` folder! Just deleted it and updated the included script path in `chat.jsp`